source/deployment/deployment.md

```diff
@@ -48,14 +47,37 @@ Typically a "Mattermost unreachable" error message will be displayed warning use
+- **2. Private Network or VPN with MFA (TE & EE)** - Private network deployment in 1) plus the option of accessing Mattermost using Virtual Private Network ("VPN") client supporting a secondary authentication method plus Mattermost's own authentication.  
+- **3. Private Network or VPN, including private Mobile Apps, with MFA (TE & EE)** - Deployment in 2) with privately-compiled mobile solution including both mobile applications and mobile push notification services compiled by the organization with organization-controlled keys or certificates (versus using public Mobile Apps, which use keys and certificates controlled by Mattermost, Inc.). 
+- **4. Private Network or VPN with MFA plus private Mobile Apps with EMM (TE & EE)** - Deployment in 3) with the addition of an enterprise mobility management solution such as Mobile Iron, Blackberry Dynamics or AirWatch for access to mobile apps in place of VPN clients. 
+- **5. Private Network or VPN with MFA, plus public Mobile Apps with 2FA (EE-only)** - Deployment in 2) plus use of publicly available mobile apps in iTunes or Google Play with the additional of MFA available in Enterprise Edition via single-sign-on providers like Okta, OneLogin, and SAML-based options, or user credentials with the addition of Google Authenticator. Recommended only for enterprises resourced to provide appropriate network security and monitoring for systems exposed to open internet. For non-enterprise deployments, VPN with MFA is recommended (see above).
+- **6. No VPN with MFA (EE-only)** - Similar to 5) with web, desktop and mobile experiences available outside private cloud with MFA enabled via Mattermost Enterprise Edition. Recommended only for enterprises resourced to provide appropriate network security and monitoring for systems exposed to open internet. For non-enterprise deployments, VPN with MFA is recommended (see above).
+- **7. No VPN, no 2FA (TE & EE)** - Recommended only for non-confidential, unimportant conversations where impact of a compromised system is not material. 
+
+### VPN Setup
+
+Mattermost is designed to offer "modern communication behind your firewall". The standard setup intended to be installed within a private network which can offer multiple factors of authentication, including the securing of access to computing devices as well as the ability to secure physical locations. 
+
+If outside access is required a virtual private network client (VPN), such as [OpenVPN](https://openvpn.net/), with an additional factor of authentication should be used to connect to Mattermost for web, desktop and mobile experiences. 
+
+### Non-VPN Setup 
+
+If a business owner decides Mattermost should be accessible from the open internet, by allowing unauthenticated access to ports such 80 and 443, the following is recommended: 
```

Comment:
Mentioning ports or the business owner doesn't seem necessary here. "If Mattermost is accessible from the open internet, the following is recommended:" makes this a bit cleaner